python/dataproc_templates/kafka/kafka_to_gcs.py

```diff
@@ -0,0 +1,117 @@
+from typing import Dict, Sequence, Optional, Any
+from logging import Logger
+import argparse
+import pprint
+
+
+
+
+from pyspark.sql import SparkSession, DataFrame
+
+
+from dataproc_templates import BaseTemplate
+import dataproc_templates.util.template_constants as constants
+
+__all__ = ['KafkaToGCSTemplate']
+
+class KafkaToGCSTemplate(BaseTemplate):
+    """
+    Dataproc template implementing loads from Kafka into GCS
+    """
+    @staticmethod
+    def parse_args(args: Optional[Sequence[str]] = None) -> Dict[str, Any]:
+        parser: argparse.ArgumentParser = argparse.ArgumentParser()
+
+        parser.add_argument(
+            f'--{constants.KAFKA_GCS_CHECKPOINT_LOCATION}',
+            dest=constants.KAFKA_GCS_CHECKPOINT_LOCATION,
+            required=True,
+            help='Checkpoint location for Kafka to GCS Template'
+        )
+        parser.add_argument(
+            f'--{constants.KAFKA_GCS_OUTPUT_LOCATION}',
+            dest=constants.KAFKA_GCS_OUTPUT_LOCATION,
+            required=True,
+            help='GCS location of the destination folder'
+        )
+        parser.add_argument(
+            f'--{constants.KAFKA_GCS_BOOTSTRAP_SERVERS}',
+            dest=constants.KAFKA_GCS_BOOTSTRAP_SERVERS,
+            required=True,
+            help='Kafka topic address from where data is coming'
+        )
+        parser.add_argument(
+            f'--{constants.KAFKA_TOPIC}',
+            dest=constants.KAFKA_TOPIC,
+            required=True,
+            help='Kafka Topic Name'
+        )
+        parser.add_argument(
+            f'--{constants.KAFKA_STARTING_OFFSET}',
+            dest=constants.KAFKA_STARTING_OFFSET,
+            required=True,
+            help='Starting offset value (earliest, latest)'
```

Comment:
It supports providing json string. Please update the help section accordingly.

Reply:
Updated